src/Corax/IndexEntryBuilder.cs

```diff
@@ -842,12 +916,28 @@ public bool Read(int field, out IndexEntryFieldType type, out Span<byte> value,
                 }
                 else if ((type & IndexEntryFieldType.Raw) != 0)
                 {
+                    if (type.HasFlag(IndexEntryFieldType.HasNull))
+                    {
+                        var spanTableOffset = MemoryMarshal.Read<int>(_buffer[intOffset..]);
+                        byte* nullTablePtr = (byte*)Unsafe.AsPointer(ref MemoryMarshal.GetReference(_buffer));
+                        if (PtrBitVector.GetBitInPointer(nullTablePtr + spanTableOffset, elementIdx) == true)
+                            goto HasNull;
+                    }
```

Comment:
No need to skip the null table here?

Reply:
Null table does not exist if there are no potential nulls stored. So no need to skip. This is a pay-for-usage type of thing... No NULL, no pay.